pages/03.guides/06.misc/docs.en.md

```diff
@@ -96,10 +96,9 @@ These are only the guides for some of them.
 ### Tachiyomi
 [Access the guide for Tachiyomi here](./tachiyomi)
 
-<hr style="border:5px solid #4ac694"> </hr>
+<hr style="border:2px solid #4ac694"> </hr>
```

Comment:
We should keep the border style the same, otherwise, looks good.

Reply:
It was not consistent with the rest of the document. Check the other borders in the document.